uptane-standard.md

```diff
@@ -645,6 +645,8 @@ The inventory database MAY record other information about ECUs and vehicles. It
 
 An Uptane-compliant ECU SHALL be able to download and verify image metadata and image binaries before installing a new image and MUST have a secure way of verifying the current time, or a sufficiently recent attestation of the time.
 
+All ECUs SHOULD monitor the download speed of image metadata and image binaries to detect and respond to a slow retrieval attack. If the download is slower than a pre-defined threshold, the ECU SHOULD provide a warning.
```

Comment:
Provide a warning to whom? To other ECUs or itself?

Reply:
I updated the text to ensure the warning is sent to the director so that it can be handled off-vehicle. I'm not sure if it should also be sent to the repo where the download originates (possibly the image repo).